strategy/rfp-to-videographers.md

```diff
@@ -1,154 +1,114 @@
 [Name]
 [Studio Name]
+[Address]
 
-Dear [Name],
+Dear [Name]:
 
-**Re. Request for proposal to produce a short video commissioned by the Humane Tech Community**
+**Re: Request for proposal to produce a short video commissioned by the Humane Tech Community**
 
-The Humane Tech Community ("HTC"), founded by The Center for Humane Technology (www.humanetech.com), is a grassroots movement whose purpose is to "Re-align Technology with Humanity's Best Interests". Based upon four pillars, namely Awareness, Regulation, Participate and Realization, members of the community are actively involved in numerous initiatives aimed at challenging the status quo.
+The Humane Tech Community ("HTC"), founded by The Center for Humane Technology (www.humanetech.com), is a grassroots movement whose purpose is to "Re-align Technology with Humanity's Best Interests". Based upon four pillars---namely, Awareness, Regulation, Participation, and Realization---the community is actively involved in numerous initiatives aimed at changing the status quo and strengthening society.
 
-One of these initiatives consists of structuring, organizing and managing several campaigns to raise public awareness of the potential pitfalls of rapid, drastic technology adoption and its significant impact on our society.
+One of these initiatives consists of structuring, organizing, and implementing several campaigns to raise public awareness of the potential pitfalls of rapid, drastic technology adoption and its significant impact on our society.
 
-The first campaign, named "Reality Shock", involves the production and distribution of a series of short, dramatic videos (each a few minutes long) aimed at shocking the audience into introspection and reflection on the new hazards presented by technology (e.g. smartphone and social media addictions, privacy, social credits, etc) and how they may be personally affected.
+The first campaign, named "Reality Shock," involves the production and distribution of a series of short, dramatic videos (each a few minutes long) aimed at shocking the audience into introspection and reflection on the hazards presented by technology (e.g., smartphone and social media addictions, privacy invasion, social-credit systems, etc.) and how people may be affected to their detriment.
 
-We believe that to achieve maximum distribution and impact, these videos should be produced by a professional studio. We would like to invite your company to submit a detailed proposal for the production of one of these videos, with the working title "A Coffee Place".
+We believe that to achieve maximum distribution and impact, these videos should be produced by a professional studio. We would like to invite your company to submit a detailed proposal for the production of one of these videos, _A Coffee Place_ (working title).
```

Comment:
```suggestion
We believe that to achieve maximum distribution and impact, these videos should be produced by a professional studio. We would like to invite your company to submit a detailed proposal for the production of one of these videos, _"Caffeine Injection"_ (working title).
```

Title had changed since creating RFP document.